util/intstrutil/util.go

```diff
@@ -23,6 +23,15 @@ func Int(is *intstr.IntOrString) (*int, error) {
 	return &i, nil
 }
 
+func Int32(is *intstr.IntOrString) (*int32, error) {
```

Comment:
I would like to add a new func to the as the standard `intstr.Parse()` creates `intstr.IntOrStr` not `*intstr.IntOrStr` which is what we always need